win32ss/gdi/eng/bitblt.c

```diff
@@ -255,6 +260,12 @@ CallDibBitBlt(SURFOBJ* OutputObj,
         psoPattern = NULL;
     }
 
+    /* Make WellOrdered with top < bottom and left < right */
+    RECTL_vMakeWellOrdered(&BltInfo.DestRect);
+
+    DPRINT("CallDibBitBlt: dstRect: (%d,%d)-(%d,%d)\n",
+      BltInfo.DestRect.left, BltInfo.DestRect.top, BltInfo.DestRect.right, BltInfo.DestRect.bottom);
```

Comment:
```suggestion
    DPRINT("CallDibBitBlt: dstRect: (%d,%d)-(%d,%d)\n",
         BltInfo.DestRect.left, BltInfo.DestRect.top, BltInfo.DestRect.right, BltInfo.DestRect.bottom);
```

Reply:
@HBelusca, This does not fit the pattern of having the lines following a DPRINT line up just after the "(" so I am doing that here hoping that this was just an oversight on your part. Thanks.

Reply:
Dunno but I was applying the generic ReactOS code style here (arguments aligned like that), not the broken formatting of win32k.

Reply:
Fixed.